_pages/en_US/troubleshooting.txt

```diff
@@ -525,6 +562,14 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>My device gives me an exception screen when booting/loading an app</u></summary>
```

Comment:
"My device gives me an" is not necessary, but might be useful/kept for clarity
app -> application